addons/ofxiOS/src/sound/SoundInputStream.m

```diff
@@ -116,43 +116,19 @@ - (void)start {
 	AVAudioSession * audioSession = [AVAudioSession sharedInstance];
 	NSError * err = nil;
 	
-    #ifdef __IPHONE_6_0
 	// need to configure set the audio category, and override to it route the audio to the speaker
 	if([audioSession respondsToSelector:@selector(setCategory:withOptions:error:)]) {
 		// we're on iOS 6 or greater, so use the AVFoundation API
 		if(![audioSession setCategory:AVAudioSessionCategoryPlayAndRecord
-						  withOptions:AVAudioSessionCategoryOptionDefaultToSpeaker
+						  withOptions:AVAudioSessionCategoryOptionMixWithOthers
 								error:&err]) {
```

Comment:
I found the issue you had before! https://github.com/openframeworks/openFrameworks/pull/2187 Seems to be related to that withOptions:AVAudioSessionCategoryOptionDefaultToSpeaker not been in iOS 5. 

In regards to what does it mean for capturing exclusive audio out... I'm not entirely sure, I think it should be fine, mix with others just allows that whole multi audio app design in iOS 9. If a developer wanted to silence that, they can with a AudioSession command regardless. So I think it should be fine.

This is the internals of the commands: tvOS prohibited so didn't have much option unless this command is really important then I could pre-processor around it if needed.
```
typedef NS_OPTIONS(NSUInteger, AVAudioSessionCategoryOptions)
{
	/* MixWithOthers is only valid with AVAudioSessionCategoryPlayAndRecord, AVAudioSessionCategoryPlayback, and  AVAudioSessionCategoryMultiRoute */
	AVAudioSessionCategoryOptionMixWithOthers			= 0x1,
	/* DuckOthers is only valid with AVAudioSessionCategoryPlayAndRecord, AVAudioSessionCategoryPlayback, and AVAudioSessionCategoryMultiRoute */
	AVAudioSessionCategoryOptionDuckOthers				= 0x2,
	/* AllowBluetooth is only valid with AVAudioSessionCategoryRecord and AVAudioSessionCategoryPlayAndRecord */
	AVAudioSessionCategoryOptionAllowBluetooth	__TVOS_PROHIBITED		= 0x4,
	/* DefaultToSpeaker is only valid with AVAudioSessionCategoryPlayAndRecord */
	AVAudioSessionCategoryOptionDefaultToSpeaker __TVOS_PROHIBITED		= 0x8,
	/* InterruptSpokenAudioAndMixWithOthers is only valid with AVAudioSessionCategoryPlayAndRecord, AVAudioSessionCategoryPlayback, and AVAudioSessionCategoryMultiRoute */
	AVAudioSessionCategoryOptionInterruptSpokenAudioAndMixWithOthers NS_AVAILABLE_IOS(9_0) 	= 0x11
} NS_AVAILABLE_IOS(6_0);
```